css/css-flexbox/flexbox_mix-writing-flex-alignment.html

```diff
@@ -0,0 +1,122 @@
<br>flex-start</div></div><div class="container vertical-lr flex-end"><div class="inner">vertical-lr<br>flex-end</div></div><div class="container vertical-lr row"><div class="inner">vertical-lr<br>row</div></div><div class="container vertical-lr row-reverse"><div class="inner">vertical-lr<br>row-reverse</div></div><div class="container vertical-lr column"><div class="inner">vertical-lr<br>column</div></div><div class="container sideways-rl column-reverse"><div class="inner">sideways-rl<br>column-reverse</div></div><div class="container sideways-rl start"><div class="inner">sideways-rl<br>start</div></div><div class="container sideways-rl center"><div class="inner">sideways-rl<br>center</div></div><div class="container sideways-rl space-around"><div class="inner">sideways-rl<br>space-around</div></div><div class="container sideways-rl space-between"><div class="inner">sideways-rl<br>space-between</div></div><div class="container sideways-rl space-evenly"><div class="inner">sideways-rl<br>space-evenly</div></div><div class="container sideways-rl stretch"><div class="inner">sideways-rl<br>stretch</div></div><div class="container sideways-rl center"><div class="inner">sideways-rl<br>center</div></div><div class="container sideways-rl flex-start"><div class="inner">sideways-rl<br>flex-start</div></div><div class="container sideways-rl flex-end"><div class="inner">sideways-rl<br>flex-end</div></div><div class="container sideways-rl row"><div class="inner">sideways-rl<br>row</div></div><div class="container sideways-rl row-reverse"><div class="inner">sideways-rl<br>row-reverse</div></div><div class="container sideways-rl column"><div class="inner">sideways-rl<br>column</div></div><div class="container sideways-lr column-reverse"><div class="inner">sideways-lr<br>column-reverse</div></div><div class="container sideways-lr start"><div class="inner">sideways-lr<br>start</div></div><div class="container sideways-lr center"><div class="inner">sideways-lr<br>center</div></div><div class="container sideways-lr space-around"><div class="inner">sideways-lr<br>space-around</div></div><div class="container sideways-lr space-between"><div class="inner">sideways-lr<br>space-between</div></div><div class="container sideways-lr space-evenly"><div class="inner">sideways-lr<br>space-evenly</div></div><div class="container sideways-lr stretch"><div class="inner">sideways-lr<br>stretch</div></div><div class="container sideways-lr center"><div class="inner">sideways-lr<br>center</div></div><div class="container sideways-lr flex-start"><div class="inner">sideways-lr<br>flex-start</div></div><div class="container sideways-lr flex-end"><div class="inner">sideways-lr<br>flex-end</div></div><div class="container sideways-lr row"><div class="inner">sideways-lr<br>row</div></div><div class="container sideways-lr row-reverse"><div class="inner">sideways-lr<br>row-reverse</div></div><div class="container sideways-lr column"><div class="inner">sideways-lr<br>column</div></div></div>
```

Comment:
Let's not hard-code this generated string; it's long and unwieldy and hard to verify all combinations are tested. I'd much rather just generate this dynamically with the code below.